src/index.js

```diff
@@ -374,7 +374,10 @@ class Dropzone extends React.Component {
       accept,
       disabled,
       type: 'file',
-      style: { display: 'none' },
+      style: {
+        display: 'none',
```

Comment:
Should it instead remove `display: none` and use a different hiding technique?

Reply:
The only technique of hiding I found that still pops up the required notifier is `opacity: 0.0001` and `pointer-events: none`. At least in chrome.

Then I set a 
```css
position: absolute;
top: 0;
left: 0;
bottom: 0;
right: 0;
```
to align the required warning to the dropzone.

What do you think?

Reply:
I believe @matuzo knows a better trick.

Reply:
Please check out https://github.com/h5bp/html5-boilerplate/blob/7e74f27208ef6e62cda92057dfb853b51d3649dd/src/css/main.css#L121-L140

Reply:
@okonet it shows the tooltip, but it doesn't work with the positioning of the tooltip, in some browsers the tooltip is down, other up (safari), this is an example on safari:
![image](https://user-images.githubusercontent.com/7217420/40928963-681834da-6823-11e8-8eb4-004513c69d28.png)

With my solution the appear fine because of the position: absolute.

Reply:
AFAIK the solution I proposed is battle tested in all modern browsers. In which browsers did you tests yours?

Reply:
@okonet Chrome, safari and firefox. I have no easy access to edge but I'm sure it works just fine

Reply:
@marcofugaro okay let's go with your for now and see if people will report issues :)

Reply:
Sure thing!